lib/bolt/transport/ssh/connection.rb

```diff
@@ -135,7 +137,7 @@ def disconnect
           end
         end
 
-        def handled_sudo(channel, data)
+        def handled_sudo(channel, data, stdin = nil)
```

Comment:
Is there a case where we _don't_ pass `stdin`? Should this be a required argument?

Reply:
There is no case where we do not pass this param. It should be required.